app/assets/appfilter.xml

```diff
@@ -2402,6 +2402,7 @@
   <item component="ComponentInfo{de.crysxd.octoapp/de.crysxd.octoapp.MainActivity}" drawable="octoapp" name="OctoApp" />
   <item component="ComponentInfo{com.gh4a/com.gh4a.activities.home.HomeActivity}" drawable="octodroid" name="OctoDroid" />
   <item component="ComponentInfo{com.kabacon.octoremote/com.kabacon.octoremote.view.activity.MainActivity}" drawable="octoremote" name="OctoRemote" />
+  <item component="ComponentInfo{com.oplus.dialer/com.oplus.dialer/com.android.contacts.DialtactsActivityAlias}" drawable="google_phone" name="ODialer" />
```

Comment:
Hey, thanks for contributing!

Just to confirm, is this Component info correct? Could be an usual one but there's usually only one `/` in it. Should the first `com.oplus.dialer` be in it?

Reply:
fixed it